hack/ocp-images.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+set -e
+set -x
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+
+# This script synchronizes OCP images into a cluster-local registry of an
+# OpenShift cluster, and writes the appropriate configuration into the
+# cluster-monitoring-operator manifests in `manifests/`.
+
+SRC_REGISTRY=registry.reg-aws.openshift.com:443/openshift3
```

Comment:
do we need to specify 443 here? and why not on `DST_REGISTRY`?

Reply:
Good point, let me check if that makes a difference.

Reply:
It is not. Updating.